readme.md

```diff
@@ -1,40 +1,183 @@
 # Objects and Functions
 
-## Learning Objectives (5 / 5)
+[![Build Status](https://travis-ci.org/ga-wdi-lessons/js-objects-functions.svg?branch=master)](https://travis-ci.org/ga-wdi-lessons/js-objects-functions)
 
-### Objects
-- Compare objects and key-value stores to arrays as data structures.
-- Explain the difference between object properties and methods.
-- Create objects using object literal syntax.
-- Compare adding and retrieving properties to objects using dot and bracket notation.
-- Use `delete`.
-- Iterate over the keys of an object to return and manipulate values.
-- Explain nested data structures.
-- Write an object method.
+## Learning Objectives
 
 ### Functions
 - Describe what a JavaScript function is.
 - Recognize the parts of a function.
 - Write a function in JavaScript using a declaration and an expression.
-- Define hoisting.
-- Differentiate between referencing and invoking a function.
 - State the difference between a function's output and side effects.
+- Differentiate between referencing and invoking a function.
+- Define hoisting.
 
-### What have you learned so far in Javascript?
+### Objects
+- Compare objects and key-value stores to arrays as data structures.
+- Create objects using object literal syntax.
+- Practice interacting with properties of literal objects.
+- Explain nested data structures.
+- Explain the difference between object properties and methods.
+- Write an object method.
+
+### Framing (15 / 15)
+**What have you learned so far in Javascript?**
 
 * Data Types
-* Data Collections
 * Conditionals
+* Data Collections
 
-## Arrays Review (5 / 10)
+## Functions
 
-Arrays are a complex data type - usually referred to as an ordered list.
+**What’s a function?**
 
-**How do we retrieve elements from an array?**
+* Fundamental component of Javascript.
+* A reusable block of Javascript code.
+* Simply put, a function is a block of code that takes an input, process that input and then produces an output.
+* Analogy: Quizno's Oven
 
-Using an index.
+**Q. Think back to what we have covered so far, when have you used functions before?**
 
-## Objects (5 / 15)
+### Why do we use functions?
+<!--
+Say we wanted the square of a number without using the above function. How would we do that?
```

Comment:
yea that was already in there. I commented it out so it wouldn't show, that's all!